### PR TITLE
install_script: env var to manually set the keys server to use.

### PR DESCRIPTION
**Pull request by remeh**
### What does this PR do?

- Adds the ability to manually set the keys server to use
- Changed the default backup key server to a static one instead of one in the SKS keyservers pool.

I've kept the `USE_APT_BACKUP_KEYSERVER` flag in order to have an easy solution to provide without having to explain what is a keys server for people just wanting to backup on another one.

### Motivation

At the moment of this PR, the SKS keyservers pool sometimes returns servers not responding.

### Additional Notes

Everything is done while declaring `keyserver` at the top of the file instead of having a condition in the middle of the code.

### CODE REVIEW

**Review by derekwbrown — CHANGES_REQUESTED** [review]
So, a couple of things
1) I'd probably prefix with `DD` just for consistency.
2) Why two variables?  If `DD_APT_BACKUP_KEYSERVER` is set, use it's value as the backup keyserver?  No need to have two different variables..

**Comment by remeh** [comment]
1. Consider it done.
2. I've done this because the user may just have a problem with the Ubuntu keys server without knowing what to use a backup keys server. By doing that both methods (manually setup one and just having a "fallback") are easy to do.
    No strong opinion on this, but it has already been done this way in puppet and ansible (where we may want to change the backup server configured...).

**Review by olivielpeau — COMMENTED** [review]
I agree with @derekwbrown that we should aim for the solution that makes the most sense for the install script, even if that means we don't support the exact same variable name/usage as what's used in puppet or ansible.

Typically, here, using one variable (`DD_KEYSERVER`) sounds like the simplest and most flexible solution. We could suggest a backup keyserver in a comment on that variable for example, if you think that could help.

File: cmd/agent/install_script.sh [hunk 1]
```diff
@@ -79,9 +79,11 @@ if [ -n "$DD_UPGRADE" ]; then
   dd_upgrade=$DD_UPGRADE
 fi
 
-use_apt_backup_keyserver=
-if [ -n "$USE_APT_BACKUP_KEYSERVER" ]; then
-  use_apt_backup_keyserver=$USE_APT_BACKUP_KEYSERVER
+keyserver="hkp://keyserver.ubuntu.com:80"
+# use this env var to specify another keys server, such as
```
Comment (olivielpeau) [inline cmd/agent/install_script.sh]: nit: `key server`

**Review by olivielpeau — DISMISSED** [review]
LGTM, just a small nit, feel free to merge once addressed

Also, forgot to mention that in general we should ensure backward compatibility when making changes to the options of the install script. However in this specific case there's a very low chance the `USE_APT_BACKUP_KEYSERVER` is actually used by anyone at the moment, so it sounds safe to remove it.

**Pull request merged**